modules/prokka/prokka.nf

```diff
@@ -17,7 +17,13 @@ process PROKKA {
     script:
 
     """
-    prokka --outdir ${genomeName} --prefix $genomeName --cpus ${task.cpus} ${bestContig} 
+    prokka --outdir ${genomeName} --prefix ${genomeName} ${bestContig} --cpus ${task.cpus}
+    """
+
+    stub:
+    """
+    echo "prokka --outdir ${genomeName} --prefix ${genomeName} ${bestContig} --cpus ${task.cpus}"
+    mkdir ${genomeName}
```

Comment:
I had to update the contents of prokka a bit further therefore this change wasn't necessary to include.